config/legacy_static_images.json

```diff
@@ -1,11 +1,20 @@
 [
   {
-    "updated": "2023-02-09",
-    "image": "us.gcr.io/broad-dsp-gcr-public/terra-jupyter-bioconductor:2.1.9",
+    "updated": "2023-06-23",
```

Comment:
Ohh I think I might have forgotten to update these last time, but it is a legacy_static  file so do we need to do this? Regardless it should be 2.1.11 for the bioconductor image I think

Reply:
oh yes definitely 2.1.11! and updating these to the most recent version before the python 3.10 release was something Alessandro suggested so that users can select those if they still want 3.7. Also in the CONTRIBUTING.md it says:

> The version of these legacy images is the version prior to a major or minor version bump